### PR TITLE
User Settings page

### PR DESCRIPTION
**Pull request by CollinFoord**
Added the User Settings page to the EHR Dashboard.

### CODE REVIEW

File: imports/ui/pages/UserSettings.jsx
Comment (jengledow) [inline imports/ui/pages/UserSettings.jsx:1]: I feel like to keep everything in line using the daisyui button and input components would be good - just as easy as adding the 'btn' and 'input' class to the respective elements

https://daisyui.com/components/input/

**Review by jengledow — COMMENTED** [review]
i put a comment on the jsx about using the daisyui components so that way it matches with the rest of the app

**Review by lattln — CHANGES_REQUESTED** [review]
image

Make sure to use the constants inside of ui/constants/dashboard.js
image

And UserSetting,jsx should be inside of ui/components/Dashboard inside of pages. Since it is a component of the "Dashboard" Page.

Looks good otherwise.

**Review by jengledow — APPROVED** [review]
besides what lin said the changes look good

**Comment by lattln** [comment]
```javascript 


export const USER_INFO = {
    name: {
        label: "Name",
        firstName_value: "John",
        lastName_value: "Smith"
    },
    gender: {
        label: "Gender",
        value: "Male",
    },
    birth: {
        label: "Birthday",
        value: "3/12/1987",
    },
    physician: {
        label: "Physician",
        value: "Dr. Dre",
    },
    lastAppt: {
        label: "Last Appointment",
        value: "10/18/2024",
    },
};


//EXAMPLE USAGE

<input
   type="text"
   id="firstName"
   name="firstName"
   value={USER_INFO.name.firstName_value}
   onChange={handleChange}
   className="input input-bordered w-full"
/>
```

**Review by lattln — APPROVED** [review]
lgtm

**Pull request merged**